addons/api/addon/abilities/collection.js

```diff
@@ -29,7 +29,8 @@ export default class CollectionAbility extends Ability {
    * @return {boolean}
    */
   hasAuthorizedCollectionAction(action) {
-    const { authorized_collection_actions } = this.model;
+    const authorized_collection_actions =
+      (this.model || {}).authorized_collection_actions || {};
```

Comment:
Thoughts on using optional chaining and using possible `undefined` return value to check if `action` include should be verified?
```
const authorized_collection_actions = this?.model?.authorized_collection_actions
```